docs/tutorial.md

```diff
@@ -0,0 +1,122 @@
+```
+This will compile the URDF from the XACRO file and save it as model.urdf in the same directory. If your launch file added parameters to the XACRO calls, you need to add them here too.
+
+<br /> 
+<br /> 
+
+## Converting the URDF to a PROTO file
+
+Convert the model.urdf with the following command. I recommend the 2 added parameters:
+**--box-collision**: simplifies objects. This can greatly improve the simulation of object interactions, especially grasping.
+
+**--multi-file**: puts the mesh data in a separate file. If not set, the proto file becomes huge and very slow and sluggish in all editors / IDEs I tried.
+
+**--static-base** and **--tool-slot=tool0** are more specific for robotic arms. Have a look at all options and explanations below. To figure out, what the **--tool-slot=LinkName** is called for your robotic arm, you will have to open the model.urdf and figure out what the link is called.
+
+```
+python -m urdf2webots.importer --input=model.urdf --box-collision --multi-file --static-base --tool-slot=tool0
+```
+
+### The script accepts the following arguments:
+  - **-h, --help**: Show the help message and exit.
+  - **--input=INFILE**: Specifies the URDF file to convert.
+  - **--output=OUTFILE**: Specifies the name of the resulting PROTO file.
+  - **--normal**: If set, the normals are exported if present in the URDF definition.
+  - **--box-collision**: If set, the bounding objects are approximated using boxes.
+  - **--disable-mesh-optimization**: If set, the duplicated vertices are not removed from the meshes (this can speed up a lot the conversion).
+  - **--multi-file**: If set, the mesh files are exported as separated PROTO files.
+  - **--static-base**: If set, the base link will have the option to be static (disable physics)
+  - **--tool-slot=LinkName**: Specify the link that you want to add a tool slot to (exact link name from URDF).
+
+<br /> 
+After your file has been converted, you should have something like this:
+
+![converted files](./images/converted_files.png)
+
+Copy these files to your project’s `protos` directory (of course you can do this step by using your OS's GUI):
+
+```
+cp -r KukaLbrIiwa14R820* ~/my_simulation/protos/
+```
+
+
+## Loading converted model in WEBOTS
+
+Launch Webots and your project world. Click on the plus sign to add your model.
+
+![add node](./images/webots_gui_1.png)
+
+You should see your newly converted model under `PROTO nodes (Current Project)`.
+Select it and click `Add`
+
+![add model](./images/webots_gui_2.png)
+
+It should look similar to this:
+
+![add model](./images/webots_robot_sideways.png)
+
+
+As you can see, there are a few things we need to fix. We want the robot to stand upright and perhaps have a fixed base by default. To make these changes, right-click on the robot in the Scene Tree and select `View PROTO Source`.
+This should open the PROTO file in Webot’s text editor and should look something like this:
+
+![add model](./images/kuka_proto.png)
```

Comment:
The two last arguments are not well aligned. Did you edited them manually or is it an issue of the script?